.github/workflows/slack_notify_p1.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 The Actions Ecosystem Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
If you make this copyright header look like one of the others where we've derived code from existing Apache 2 stuff then you won't need to exempt it from the copyright check. See e.g.:

https://github.com/MaterializeInc/materialize/blob/9078e8b4882165556045982d055b73e78a9090b9/src/ore/src/netio/read_exact.rs#L1-L23

tl;dr we get to slap our copyright on it which applies to any modifications we make, but leave the Apache license and original copyright in place.

Reply:
Gotcha! I wasn't sure how to handle the licensing. Will adapt it to match the example you shared!